electron/settingsRenderer.js

```diff
@@ -250,6 +251,22 @@ rivets.binders.deckid = function(el, value) {
   el.setAttribute('data-deckid', value);
 }
 
+rivets.binders.mythicprogress = function(el, value) {
+  el.style.width = Math.max(10, (100 * value.mythicOwned / value.mythicTotal)) + "%"
```

Comment:
Why default of 10%?

Reply:
If you don't set some mimimum width, then very small percentages will render very poorly:

![image](https://user-images.githubusercontent.com/1424566/48962780-1f798780-ef3a-11e8-9aba-7241a651eda3.png)

Unfortunately, this has a side effect of misrepresenting 0 collected rows (it looks like you have collected some), but imo the `0 / 123 Mythics` should make that trade-off ok. What do you think?

Reply:
Maybe do a little gymnastics to move the "X/Y Rarity" to the other part of the bar if the collected is too small? Maybe shrink the bars, putting the percentage on the collected if large enough and "X/Y" at the end after the bar?

Reply:
Good idea. How does this look?

![image](https://user-images.githubusercontent.com/1424566/48963045-80f02500-ef3f-11e8-8d85-813c0802e73d.png)

(commit inc.)

Reply:
There we go.